modules/claims_api/app/controllers/claims_api/v2/veterans/evidence_waiver_controller.rb

```diff
@@ -4,14 +4,18 @@
 require 'token_validation/v2/client'
 require 'claims_api/claim_logger'
 require 'claims_api/dependent_service'
+require 'claims_api/v2/params_validation/evidence_waiver_submission'
 
 module ClaimsApi
   module V2
     module Veterans
       class EvidenceWaiverController < ClaimsApi::V2::Veterans::Base
         skip_before_action :validate_json_format
+        FORM_NUMBER = '5103'
 
         def submit
+          validate_request!(ClaimsApi::V2::ParamsValidation::EvidenceWaiverSubmission)
+
```

Comment:
@stiehlrod This appears to be reverting [#17413](https://github.com/department-of-veterans-affairs/vets-api/pull/17413) (oh probably related to the conflicts below, didn't see those at first)